### DISCUSSION TITLE
Youtube Music 32 bit

### DISCUSSION THREAD

**Discussion by wadshrad** [ReVanced/revanced-discussions#1052] [Question] [answered]
I'm getting a warning that Youtube music and Youtube music extended is 32 bit when I open it. 
I've tried uninstalling and reinstalling both from revanced manager. 
Does anyone know if both architectures are support and how to get it to 64 bit install if so?

![image](https://github.com/user-attachments/assets/398ecd6d-2f9e-4902-990e-2707caf2df9e)

**Comment by oSumAtrIX** [comment, marked as answer]
You need to find 64 bit versions online. And this is ReVanced and not extended.

**Reply by wadshrad** [reply]
Thanks, I tried both Revanced and Extended. 
I'd been getting this for a while. Of course as soon a I reach out about it I also update to the latest version and now the warning has gone. 
🙄
Thanks for the quick response.

**Comment by sammy76** [comment]
Love it 😐🤪
